recipes/magic-wormhole/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "magic-wormhole" %}
+{% set version = "0.9.1" %}
+{% set sha256 = "c4471b5dfb5f802024659553974f2bd414b690f907d63a4cdc17d0caba112172" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or py2k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+
+  run:
+    - python
+    - twisted
+    - humanize
+    - click
+    - tqdm
+    - spake2 ==0.7
+    - pynacl
+    - six
+    - twisted
+    - autobahn >=0.14.1
+    - hkdf
+    - pip
+
+
+test:
+  commands:
+    - wormhole --version
+
+
+about:
+  home: http://github.com/simplejson/simplejson
```

Comment:
I really need to write a piece of the linter for catching things from the examples.